Extension/src/common.ts

```diff
@@ -123,7 +123,7 @@ export async function ensureBuildTaskExists(taskName: string): Promise<void> {
         throw new Error("Failed to get tasksJsonPath in ensureBuildTaskExists()");
     }
 
-    await writeFileText(tasksJsonPath, JSON.stringify(rawTasksJson, null, settings.editorTabSize));
```

Comment:
Text "// TODO: It's dangerous to overwrite this file. We could be wiping out comments." above should be removed.